### ISSUE TITLE
What version of discord-py and aiohttp do I use?

### ISSUE THREAD

**Issue by Avowned** [nekitdev/gd.rpc#5]
When installing gd-py with python, I get this error:
'discord-py 1.7.3 has requirement aiohttp<3.8.0,>=3.6.0, but you'll have aiohttp 3.8.6 which is incompatible.'

So then, I downgrade aiohttp to 3.8.0, but it says:
'gd-py 1.0.0 has requirement aiohttp>=3.8.4, but you'll have aiohttp 3.8.0 which is incompatible.
discord-py 1.7.3 has requirement aiohttp<3.8.0,>=3.6.0, but you'll have aiohttp 3.8.0 which is incompatible.'

Basically I don't know which are the compatible versions and I keep installing unnecessary files.
